the-collab-lab/tcl-16-smart-shopping-list
# Resolves 25. As a search engine, I want to see metadata for me to understand what MCB is about

## Description

This PR updates our metadata so that search engines understand what MCB is by updating meta tags, favicon, and `manifest.json`. Additionally I added meta tags for the [Open Graph protocol](https://ogp.me) so that social sites (ex: Facebook and LinkedIn) will know how to display information about MCB.

## Related Issue

closes #55 

## Acceptance Criteria

Update the following meta tags:
- page title
- favicon
- description
- Open Graph (image, title, description, url, and type)

Helpful resources to reference:
- https://www.w3schools.com/tags/tag_meta.asp
- https://neilpatel.com/blog/open-graph-meta-tags/
- https://ogp.me/
- https://medium.com/@weberzt/how-to-add-a-display-image-to-your-react-app-using-meta-tags-1b84fd50f57

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


<img width="235" alt="Screen Shot 2021-05-14 at 10 12 56 AM" src="https://user-images.githubusercontent.com/62153993/118283115-057ee700-b49d-11eb-81c7-237e4940b2d2.png">

### After


<img width="235" alt="Screen Shot 2021-05-14 at 10 13 18 AM" src="https://user-images.githubusercontent.com/62153993/118283189-16c7f380-b49d-11eb-89f8-69bdc9d3f8cf.png">

## Testing Steps / QA Criteria



* switch to this branch `git checkout metadata`
* `npm start`
* when navigating to the browser, check if the updated favicon and title appear
* (optional) check on localhost if the open graph data is updated
